### PR TITLE
Add s390x support for travis

### PR DESCRIPTION
**Pull request by srajmane**
As Travis CI [officially supports](https://blog.travis-ci.com/2019-11-12-multi-cpu-architecture-ibm-power-ibm-z) s390x builds, adding support for same.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,12 @@ sudo: required
 services:
   - docker
 
+jobs:
+  include:
+   - arch: amd64
+   - arch: s390x
+     dist: bionic
```
Comment (SuperSandro2000) [inline .travis.yml:11]: I don't think we should test one on trusty and one on bionic. Tests could error because of differences between those two.

Comment (srajmane) [reply .travis.yml:11]: s390x build runs within an LXD container on Z-based infrastructure. LXD containers are with (Bionic), Xenial only.
Reference: https://docs.travis-ci.com/user/reference/overview/

Comment (srajmane) [reply .travis.yml:11]: @SuperSandro2000 Could you please have a look ?

Comment (SuperSandro2000) [reply .travis.yml:11]: I have nothing to say here but a good approach would be to update the other tests also to bionic.

Comment (srajmane) [reply .travis.yml]: @SuperSandro2000 Added dist: bionic for `amd64` and `s390x`. Also set  GO111MODULE=on to resolve `l.LintPkg undefined (type *"github.com/golangci/lint-1".Linter has no field or method LintPkg)` error.
Ref: https://github.com/golangci/golangci-lint#go

**Comment by srajmane** [comment]
@caervs  Could you please have a look ?

**Comment by srajmane** [comment]
@thaJeztah @dmcgowan Please have a look.

**Comment by manishtomar** [comment]
@thaJeztah What is our policy in general for other arch support? Do you think its sufficient to just have CI enabled?

**Comment by thaJeztah** [comment]
I don't think there's an official "policy" for this repository ( we _are_ working with the IBM Power and  Z teams on docker packages for those platforms, and we have CI running for those in the upstream moby repository)

I don't think it hurts to have CI running to check if  nothing breaks on those architectures

**Comment by thaJeztah** [comment]
Perhaps arm64 should be enabled as well if it doesn't considerably slow down testing on PR's

**Comment by srajmane** [comment]
Hi @olegburov , added s390x support to make sure nothing breaks for s390x.

**Comment by olegburov** [comment]
Sorry, I meant why does the Registry care of `s390x`? Is this architecture it must to support?

**Comment by manishtomar** [comment]
@olegburov I don't think this repo provides "official" support of `s390x` by having it in CI. If any user has problems with that arch they are welcome to create a ticket or open a PR. It is a different matter whether it will be merged or worked on. Like @thaJeztah said it doesn't hurt adding the CI validation.

**Comment by olegburov** [comment]
Got it. The reason why I asked is, if we want to switch from Travis CI to GitHub Actions in some future, we will have to drop `s390x` architecture, because it doesn't support it.

**Pull request merged**